.github/renovate.json

```diff
@@ -21,9 +21,14 @@
   },
   "packageRules": [
     {
-      "matchDepTypes": ["devDependencies"],
-      "matchUpdateTypes": ["patch"],
-      "enabled": false
+      "groupName": "major-minor dependencies grouped by manager",
+      "branchName": "{{manager}}-dependencies",
+      "matchUpdateTypes": ["major", "minor"]
```

Comment:
it might make sense to not have `major` to be grouped as it probably has breaking changes

Reply:
Should we group all minor and patch dependencies together per package manager and major updates can continue separately?

Reply:
yeah that sounds good :)

Reply:
updated accordingly, PTAL